database-collection.md

```diff
@@ -32,46 +32,108 @@ However, collections are much more powerful than arrays and expose a variety of
             return $user->name;
         });
 
-<a name="usage-examples"></a>
-## Usage examples
```

Comment:
Why remove this section?

Reply:
@LukeTowers It was renamed to "Available Methods" - more fitting for a bigger list of methods IMO.

Reply:
@bennothommo can you keep the anchor in place for existing links?

Reply:
@LukeTowers no worries - done.